storage/config.go

```diff
@@ -20,4 +20,6 @@ type Configuration struct {
 	Address string `yaml:"address"`
 	// Directory with .sql files that are run at plugin startup.
 	InitSQLScriptsDir string `yaml:"init_sql_scripts_dir"`
+	// Url for exposing prometheus metrics e.g. http://localhost:1000.
+	MetricsURL string `yaml:"metrics_url"`
```

Comment:
Could we rename it to `MetricsEndpoint`? 

endpoint is more used in OTELcol and Jaeger will be rebased on top of it soon.